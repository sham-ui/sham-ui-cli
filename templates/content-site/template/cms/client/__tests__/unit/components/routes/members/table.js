import RoutesMembersTable  from '../../../../../src/components/routes/members/table.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( RoutesMembersTable, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
