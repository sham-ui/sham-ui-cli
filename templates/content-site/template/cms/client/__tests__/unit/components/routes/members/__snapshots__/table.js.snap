// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "deleteModalVisible": false,
    "member": null,
    "members": Array [],
    "onSelectMember": [Function],
    "refresh": [Function],
    "showErrors": [Function],
  },
  "html": "
<table>
  <thead>
    <tr>
      <th>ID</th>
      <th>Name</th>
      <th>Email</th>
      <th>Is superuser?</th>
      <th></th>
    </tr>
  </thead>
  <tbody></tbody>
</table>
<!--0-->
",
}
`;
